opencti-platform/opencti-graphql/src/schema/identifier.js

```diff
@@ -328,11 +328,11 @@ const filteredIdContributions = (contrib, way, data) => {
     }
     const destKey = dest || src;
     const resolver = contrib.resolvers[src];
-    if (resolver) {
-      objectData[destKey] = value ? resolver(value) : value;
-    } else {
-      objectData[destKey] = value;
+    const resolvedValue = (resolver && value) ? resolver(value) : value;
+    if (isEmptyField(resolvedValue)) {
+      return {}; // missing contribution
     }
```

Comment:
For instance, name resolvers always lower-case and trim the given value.
We could thus end up with an empty string where the code expected a real contribution to the uuid generation. Later in the code, this contribution would be discarded.

With malwares that require only name, this would lead to error when generating the identifier.
With Reports, the published field would still be kept allowing to generate the id.

With this fix, creating a report with only space in the name should fail with an error here at identifier generation time.

```
mutation {
  reportAdd(input: { name: "      ", published: "2024-04-17T08:49:10.677Z"}) {id}
  # malwareAdd(input: { name: "     " }) {id}
}
```